resources/config/settings.yaml

```diff
@@ -124,4 +124,9 @@ edit_area_style:
         default: 14
         user: 14
 
+arabic_pane_style:
+    fontsize:
+        default: 14
```

Comment:
I'd make the arabic font default to 16-18 because it's always too small compared to the non arabic font.